--- /usr/share/jolla-alarm-ui/pages/ClockAlarmDialog.qml	2014-05-24 21:17:22.192520750 +0200
+++ /usr/share/jolla-alarm-ui/pages/ClockAlarmDialog.qml	2014-05-24 21:16:08.914323771 +0200
@@ -4,7 +4,7 @@
 import org.nemomobile.time 1.0
 
 AlarmDialogBase {
-    onTimeout: closeDialog(AlarmDialogStatus.Closed)
+    onTimeout: closeDialog(AlarmDialogStatus.Snoozed)
 
     topIconSource: "image://theme/icon-l-snooze?" + Theme.highlightColor
 
